### PR TITLE
Move to cost taking IO into account.

### PR DESCRIPTION
**Pull request by olonho**

### CODE REVIEW

File: core/primitives/src/version.rs [hunk 1]
```diff
@@ -92,7 +92,7 @@ pub const PROTOCOL_VERSION: ProtocolVersion = 41;
 
 /// Current latest nightly version of the protocol.
 #[cfg(feature = "nightly_protocol")]
-pub const PROTOCOL_VERSION: ProtocolVersion = 47;
+pub const PROTOCOL_VERSION: ProtocolVersion = 48;
```
Comment (bowenwang1996) [inline core/primitives/src/version.rs:95]: what is the new feature? We update nightly protocol version only if we change a feature or add a new feature

File: runtime/runtime-params-estimator/Cargo.toml [hunk 2]
```diff
@@ -50,7 +50,7 @@ costs_counting = ["near-vm-logic/costs_counting",
                   "testlib/costs_counting",
                   "near-evm-runner/costs_counting"]
 # Required feature for proper config, but can't be enabled by default because it is leaked to other release crates.
-required = ["costs_counting", "near-vm-runner/no_cpu_compatibility_checks", "no_cache", "near-store/single_thread_rocksdb"]
+required = ["costs_counting", "near-vm-runner/no_cpu_compatibility_checks", "no_cache"]
```
Comment (evgenykuzyakov) [inline runtime/runtime-params-estimator/Cargo.toml:53]: Still don't understand why we don't run single thread rocksdb.

**Review by evgenykuzyakov — CHANGES_REQUESTED** [review]
Changing default will create an issue and not upgrade the existing protocol. There is a PR with a proper fix using a new experimental feature that does live migration: https://github.com/near/nearcore/pull/3653

But we also have a breaking change with the gas price computation based on the total prepaid gas amount, but this PR doesn't change it.

**Comment by bowenwang1996** [comment]
@olonho do we still want to keep this PR?

**Comment by matklad** [comment]
We definitely want to wrap this work up, as the current situation is self-contradictory: #4474. The current params estimator doesn't actually the costs we currently use. Not sure if its better to revive this PR or create a new one, that's up to @olonho

**Comment by olonho** [comment]
No longer relevant.

**Pull request closed without merging**